CHANGES.md

```diff
@@ -26,9 +26,13 @@ To be released.
 ### Bug fixes
 
  -  Fixed a bug that `Swarm` reported `TaskCanceledException` as an unknown
-   exception while stopping. [[#275]]
+    exception while stopping. [[#275]]
  -  Fixed a bug that `Swarm` didn't stop properly during `Swarm.Preload()`.
-   [[#275]]
+    [[#275]]
+ -  Fixed a bug where the first `TxNonce` of an address is not stripped when
```

Comment:
What does <q>stripped</q> mean here?

Reply:
<q>strip</q> means [this](https://github.com/planetarium/libplanet/blob/03caf772deef950236387d4502bd2eed9ab23e17/Libplanet/Store/FileStore.cs#L759). To fork `TxNonce`s,  we copy and strip them.

Reply:
Term suggestions:

- the first `TxNonce` → the oldest `TxNonce`
- stripped → invalidated